homeassistant/components/cloud/manifest.json

```diff
@@ -13,6 +13,6 @@
   "integration_type": "system",
   "iot_class": "cloud_push",
   "loggers": ["hass_nabucasa"],
-  "requirements": ["hass-nabucasa==0.87.0"],
+  "requirements": ["hass-nabucasa==0.88.0"],
```

Comment:
0.88.0 contains a breaking change where the `alexa_server` Cloud parameter and attribute was removed. Our code needs to be updated to handle this.

Reply:
Additionally, as 0.88.1 is now released, this PR should be adjusted to use that one.